js/noms/src/http-batch-store.js

```diff
@@ -52,11 +52,12 @@ const readBatchOptions = {
 
 const writeBatchOptions = {
   method: 'POST',
+  headers: {
+    'Content-Type': 'application/octet-stream',
+  },
 };
 
-const updateRootOptions = {
-  method: 'POST',
-};
+const updateRootOptions = writeBatchOptions;
```

Comment:
`updateRoot` has no post form data, so it's odd to put a content type on it. Not a big deal I suppose.